src/test/resources/demo-counterexamples/cyclic-ref.vpr

```diff
@@ -0,0 +1,9 @@
+field next: Ref
```

Comment:
Shouldn't there be a copyright message at the beginning of each test file? I guess we're not very consistent here, though.